crates/holochain/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 ---
 default_semver_increment_mode: !pre_minor dev
 ---
+
 # Changelog
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+- Mark `schedule` host fn as stable in code. #5240
```

Comment:
This should be the only change in this file. Please undo the formatting.

Reply:
Fixed. Weird. I didn't see that when I reviewed the PR.